app/models/assignment/editor.rb

```diff
@@ -91,7 +91,9 @@ def update_attribute_for_all_assignment_repos(attribute:, change:)
     end
 
     def deadline_updated?
-      @options[:deadline] != @assignment.deadline&.deadline_at
+      return false unless @options[:deadline]
+      new_deadline_at = DateTime.strptime(@options[:deadline], DATETIME_FORMAT).utc
```

Comment:
I'd rather have this look something like

```ruby
new_deadline_at = DateTime.strptime(@options[:deadline], Deadline::Factory::DATETIME_FORMAT).utc
```

Instead of a constants file just hanging out as an initializer